react-native/package.json

```diff
@@ -3,13 +3,14 @@
   "main": "node_modules/expo/AppEntry.js",
   "private": true,
   "scripts": {
-    "start": "expo start",
-    "android": "expo start --android",
-    "ios": "expo start --ios",
-    "eject": "expo eject"
+    "start": "npx expo start",
+    "android": "npx expo start --android",
+    "ios": "npx expo start --ios",
+    "eject": "npx expo eject"
   },
   "dependencies": {
     "expo": "^30.0.1",
+    "expo-cli": "^2.11.9",
```

Comment:
why adding this was necessary?

Reply:
because otherwise when doing ```npm install```  installing just expo package asks to install ```expo-cli``` globally and from my knowledge, it's not a good idea, the user will need to install it manually with ```sudo```.